nomad/local/integration-tests.nomad

```diff
@@ -104,5 +111,235 @@ job "integration-tests" {
       }
     }
   }
+  group "analyzerlib-integration-tests" {
+    restart {
+      # Make this a one-shot job
+      attempts = 0
+    }
+
+    network {
+      mode = "bridge"
+    }
+
+    # Enable service discovery
+    service {
+      connect {
+        sidecar_service {
+          proxy {
+            upstreams {
+              # This is a hack, because IDK how to share locals across files
+              destination_name = "dgraph-alpha-0-grpc-public"
+              local_bind_port  = 9080
+            }
+          }
+        }
+      }
+    }
+
+    task "analyzerlib-integration-tests" {
+      driver = "docker"
+
+      config {
+        image      = "${var.container_registry}/grapl/analyzerlib-test:latest"
+        entrypoint = ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
+        command    = "cd grapl_analyzerlib && py.test -v -n auto -m 'integration_test'"
+      }
+
+      env {
+        DEPLOYMENT_NAME = var.deployment_name
+        GRAPL_LOG_LEVEL = local.log_level
+        MG_ALPHAS       = "localhost:9080"
```

Comment:
I suspect it actually needs to be that `localhost_within_bridge` but if it works it works?